tests/Unit/ParallelAlgorithms/Events/Test_ObserveAtExtremum.cpp

```diff
@@ -0,0 +1,288 @@
+      box, ActionTesting::cache<element_component>(runner, array_index),
+      array_index, std::add_pointer_t<element_component>{}));
+
+  observe->run(
+      make_observation_box<
+          tmpl::filter<typename ObserveAtExtremumEvent<
+                           ArraySectionIdTag>::compute_tags_for_observation_box,
+                       db::is_compute_tag<tmpl::_1>>>(box),
+      ActionTesting::cache<element_component>(runner, array_index), array_index,
+      std::add_pointer_t<element_component>{});
+
+  // Process the data
+  runner.template invoke_queued_simple_action<observer_component>(0);
+  CHECK(runner.template is_simple_action_queue_empty<observer_component>(0));
+
+  const auto& results = MockContributeReductionData::results;
+  CHECK(results.observation_id.value() == observation_time);
+  CHECK(results.observation_id.observation_key() ==
+        expected_observation_key_for_reg);
+  CHECK(results.subfile_name == expected_subfile_name);
+  CHECK(results.time == observation_time);
+  CHECK(results.legend[0] == "Time");
+  if (extremum_type == "Max") {
+    CHECK(results.legend[1] == "Max of Var0");
+    CHECK(results.legend[2] == "Value of Var0 at Max of Var0");
+    CHECK(results.legend[3] == "Value of Var1_x at Max of Var0");
+    CHECK(results.legend[4] == "Value of Var1_y at Max of Var0");
+    CHECK(results.legend[5] == "Value of Var1_z at Max of Var0");
+    CHECK(results.reduced_vector ==
+          std::vector<double>{27.0, 27.0, 11.0, 38.0, 65.0});
+  } else {
+    CHECK(results.legend[1] == "Minus Min of Var0");
+    CHECK(results.legend[2] == "Value of Var0 at Min of Var0");
+    CHECK(results.legend[3] == "Value of Var1_x at Min of Var0");
+    CHECK(results.legend[4] == "Value of Var1_y at Min of Var0");
+    CHECK(results.legend[5] == "Value of Var1_z at Min of Var0");
+    CHECK(results.reduced_vector ==
+          std::vector<double>{-1.0, 1.0, 28.0, 12.0, 39.0});
+  }
+}
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Evolution.ObserveAtExtremum", "[Unit][Evolution]") {
+  test<TestSectionIdTag>(
+      std::make_unique<ObserveAtExtremumEvent<TestSectionIdTag>>(
+          ObserveAtExtremumEvent<TestSectionIdTag>{
+              "reduction0", {"Var0", "Max", {"Var0", "Var1"}}}),
+      "Max", Spectral::Basis::Legendre, Spectral::Quadrature::GaussLobatto,
+      "Section0");
+  test<TestSectionIdTag>(
+      std::make_unique<ObserveAtExtremumEvent<TestSectionIdTag>>(
+          ObserveAtExtremumEvent<TestSectionIdTag>{
+              "reduction0", {"Var0", "Min", {"Var0", "Var1"}}}),
+      "Min", Spectral::Basis::Legendre, Spectral::Quadrature::GaussLobatto,
+      "Section0");
+
+  INFO("create/serialize");
+  Parallel::register_factory_classes_with_charm<Metavariables<3, void>>();
+  const auto factory_event = TestHelpers::test_creation<std::unique_ptr<Event>,
+                                                        Metavariables<3, void>>(
+      // [input_file_examples]
```

Comment:
You don't use this example anywhere.  Either add it to the docs or remove the label comments.

Reply:
I though this was used by the surrounding code to create an event (and I did get some error from those input files at some point). Isn't this the same as the input file examples in Test_ObserveNorms?

Reply:
The `test_creation` function call is good, but as-is you're not using the `// [input_file_examples]` comments.  Those are used in documentation to put example code snippets in the docs.  For example, in `ObserveNorms.hpp` in the class documentation there is
```
 * Here is an example of an input file:
 *
 * \snippet Test_ObserveNorms.cpp input_file_examples
```
to match the `// [input_file_examples]` comments in the test.  You can either add a similar "snippet" line to your docs or remove the comments here.

Reply:
Ah, got it; easy enough to add that to the documentation then.